intro/scipy/index.rst

```diff
@@ -534,71 +464,15 @@ We can constrain the variable to the interval
     Hints:
 
         - Variables can be restricted to :math:`-2 < x < 2` and :math:`-1 < y < 1`.
-        - Use :func:`numpy.meshgrid` and :func:`matplotlib.pyplot.imshow` to
-          find visually the regions.
-        - Use :func:`scipy.optimize.minimize`, optionally trying out
-          several of its methods.
-
-    How many global minima are there, and what is the function value at those
-    points?  What happens for an initial guess of :math:`(x, y) = (0, 0)` ?
+        - :func:`numpy.meshgrid` and :func:`matplotlib.pyplot.imshow` can help
+          with visualization.
+        - Try minimizing with :func:`scipy.optimize.minimize` with an initial
+          guess of :math:`(x, y) = (0, 0)`. Does it find the global minimum, or
+          converge to a local minimum? What about other initial guesses?
+        - Try minimizing with :func:`scipy.optimize.differential_evolution`.
 
     :ref:`solution <sphx_glr_intro_scipy_auto_examples_plot_2d_minimization.py>`
 
-
-Finding the roots of a scalar function
```

Comment:
I'm assuming that gh-583, which adds this to the top, will be merged first.

Reply:
gh-853 has been merged.